public-ui/src/app/data-browser/views/ehr-view/components/concept-row-react.component.tsx

```diff
@@ -0,0 +1,385 @@
+          alertBox.style.left = e.pageX - 60 + 'px';
+          alertBox.innerHTML =
+            `<div class="copy-alert">
+            ${message}
+          </div>`;
+          // alertBox.innerText = message;
+          document.body.appendChild(alertBox);
+          setTimeout(() => {
+            document.body.removeChild(alertBox);
+          }, 400);
+      }
+
+    render() {
+        const {concept, domain, totalResults, maxResults, currentPage, counter, searchTerm, synonymString} = this.props;
+        const {showMoreSynonyms, showMoreDrugBrands, showConceptChart, graphToShow} = this.state;
+        const id = 'c' + concept.conceptId;
+        const tabIndex = 0;
+        const tooltipAction = 'Concept synonyms tooltip hover on concept ' + concept.conceptName;
+        let conceptIndex = counter + 1;
+        if (totalResults > maxResults) {
+            conceptIndex = counter + ((currentPage - 1) * maxResults) + 1;
+        }
+        const synonymsStr = showMoreSynonyms ? synonymString : (synonymString ? synonymString.substring(0, 100) : null);
+        const drugBrandsStr = showMoreDrugBrands ? concept.drugBrands.join(', ') : concept.drugBrands.slice(0, 10).join(', ');
+        const tblClass = domain === 'labs & measurements' ? 'tbl-r-labs' : 'tbl-r';
+        return <React.Fragment>
+               <style>{cssStyles}</style>
+               <div id={id}>
+                 <div className='tbl-exp-r' onClick={() => this.expandRow()}>
+                 <div className={tblClass} style={domain === 'labs & measurements' ? styles.tblRLabs : styles.tblRow}>
+                 <div className='body-lead tbl-d' style={styles.bodyLead}>
+                    <span>{conceptIndex}. </span>
+                    <HighlightReactComponent searchTerm={searchTerm} text={concept.conceptName}/>
+                 </div>
+                 <div className='body-lead tbl-d' style={styles.bodyLead}>
+                    {concept.countValue <= 20 && <span>&le; </span>} {concept.countValue.toLocaleString()}
+                 </div>
+                 <div className='body-lead tbl-d' style={styles.bodyLead}>
+                    {this.participantPercentage(concept.countValue)} %
+                 </div>
+                 {domain === 'labs & measurements' && concept.measurementConceptInfo &&
+                 <div className='body-lead tbl-d'>
+                    {concept.measurementConceptInfo.hasValues === 1 ?
+                    <span className='test-span' style={styles.measurementTypeSpan}><i className='fas fa-vial' style={{'transform': 'rotate(315deg)'}}></i>
+                    <TooltipReactComponent tooltipKey='valueFilter' label='' searchTerm='' action=''></TooltipReactComponent></span> :
+                    <span className='order-span' style={styles.measurementTypeSpan}><i className='far fa-file-signature'></i>
+                   <TooltipReactComponent tooltipKey='orderFilter' label='' searchTerm='' action=''></TooltipReactComponent></span>}
```

Comment:
Nit: Should use the self-closing tag syntax for empty element tags

Reply:
Changed this thanks.